### PR TITLE
Delegate to Node for Settings + Info, remove dead code

### PR DESCRIPTION
**Pull request by parsonsmatt**
<p align="right">#9</p>

# Overview

- [x] I have modified the Info and Settings endpoints to point to the node API.
- [x] Removed the old code for these features from the wallet.

# Comments

### CODE REVIEW

File: src/Cardano/Wallet/API/Internal/Handlers.hs [hunk 1]
```diff
@@ -48,21 +45,6 @@ applyUpdate nc = do
         Right () ->
             pure NoContent
 
-handleNodeError :: ClientError () -> Handler a
```
Comment (parsonsmatt) [inline src/Cardano/Wallet/API/Internal/Handlers.hs:51]: Relocated into `NodeProxy` module.

File: src/Cardano/Wallet/API/V1/Handlers.hs [hunk 2]
```diff
@@ -11,16 +11,18 @@ import qualified Cardano.Wallet.API.V1.Handlers.Settings as Settings
...
-         :<|> Settings.handlers     passiveWallet
-         :<|> Info.handlers         w
+handlers :: NodeHttpClient -> ActiveWalletLayer IO -> Server V1.API
+handlers nodeClient w =
+    Addresses.handlers         passiveWallet
+    :<|> Wallets.handlers      passiveWallet
+    :<|> Accounts.handlers     passiveWallet
+    :<|> Transactions.handlers w
+    :<|> Settings.handlers     nodeClient
+    :<|> Info.handlers         nodeClient
```
Comment (parsonsmatt) [inline src/Cardano/Wallet/API/V1/Handlers.hs:26]: tempted to bite the bullet and just remove all the vertical alignment here

**Review by parsonsmatt — COMMENTED** [review]
This PR could be broken up into multiple PRs if you feel that's a better idea.

File: src/Cardano/Wallet/API/V1/Handlers.hs (hunk 2 above)
Comment (KtorZ) [reply src/Cardano/Wallet/API/V1/Handlers.hs:26]: I know that feel. Maybe we should allow one day per month called "alignment catharsis day" where we only submit PRs to fix variable-length vertical alignments.

**Review by KtorZ — APPROVED** [review]

**Review by akegalj — APPROVED** [review]

**Pull request merged**